### ISSUE TITLE
Collapse comments by clicking the nice big button that shows the upvote count

### ISSUE THREAD

**Issue by Welding-Torch** [spikecodes/libreddit#231]
## Is your feature request related to a problem? Please describe.

  When I first tried this site, I instinctively tried to click that button to collapse comments. I think that if I did that, there's a decent chance other people tried to. And after all, intuitive design is good design right?

## Describe the solution you'd like

  I think it'd be pretty cool if a user could just click that nice, big button to collapse that comment.

## Additional context
  Just to clarify, this is the button I'm talking about:
![Screenshot 2021-06-06 211854](https://user-images.githubusercontent.com/46340124/120930887-d14dbd80-c70c-11eb-9906-590fe03c820f.jpg)

Thank you!

**Comment by curlpipe** [comment]
This would be especially good on mobile devices, as it's a pain in the rear-end to expand and collapse comments.

**Comment by spikecodes** [comment]
Thanks for this suggestion, though I'm not sure if I understand what you mean.

Is this the functionality you are looking for?

https://user-images.githubusercontent.com/19519553/121759850-465b3180-cb17-11eb-8032-8e580e9b8673.mp4

**Comment by Mennaruuk** [comment]
I think what they want is the upvote count to work like the down arrow.

**Comment by Welding-Torch** [comment]
Yes @Mennaruuk is correct. Right now, there are two elements there - an 'upvote count button' and an 'arrow'. What I think would be cool, is combining those two into one button with the same _look_ as the upvote count button but with the _functionality_ of the arrow. I think this makes it easier to click and makes the site a bit more user friendly. 

Please let me know if you have any more questions. Thank you! You have made an amazing site.

**Comment by spikecodes** [comment]
Oh I see, that makes sense. Thank you for this suggestion!

I'm happy that you like Libreddit! :)

**Comment by mikupls** [comment]
+1 for this feature request. It seems very similar to #250 though. And if I understood correct, we are also limited by #77 here.

Seeing how we have `HLS,js`, I'm tempted to ask how strict the "no js" rule is for this project :)
I'm a very happy to having to touch any JS in this project, but I wonder if a widely-requested feature like this would make a good exception. (Especially a feature which is needed to bring libreddit on-par with vanilla Reddit usability-wise, and is seemingly impossible to fulfill with HTML/CSS only)

**Comment by Welding-Torch** [comment]
I would like to add that even though enhancement request is a couple of months old, I would still like to see it become a reality if it is possible. 

Thanks @mikupls for having a look at this and contributing very practically!
